### PR TITLE
Pending enterprise customer admin user view set permission update

### PR DESCRIPTION
**Pull request by MueezKhan246**
Description:
Added IsInProvisioningAdminGroup permission class and updated exception handling in serializer.

JIRA:
https://2u-internal.atlassian.net/jira/software/c/projects/ENT/issues/ENT-8995

### CODE REVIEW

**Review by hamzawaleed01 — APPROVED** [review]
@MueezKhan246  I know it's indirectly being checked in other tests but still can we add one more test that checks if user is part of provisioning group? just to make sure that valid exception is being raised.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -17,6 +17,10 @@ Unreleased
 ----------
 * nothing unreleased
 
+[4.19.8]
+* feat: updated exception handling in serializer
+* feat: added is_in_provisioning_admin_group permission class
```
Comment (zamanafzal) [inline CHANGELOG.rst:22]: Please update this to something which is more precise and do not use feat.

**Review by zamanafzal — APPROVED** [review]

**Comment by MueezKhan246** [comment]
> @MueezKhan246 I know it's indirectly being checked in other tests but still can we add one more test that checks if user is part of provisioning group? just to make sure that valid exception is being raised.

@hamzawaleed01 i have updated `test_post_pending_enterprise_customer_unauthorized_user` for this, kindly check.

File: CHANGELOG.rst (hunk 1 above)
Comment (MueezKhan246) [reply CHANGELOG.rst]: @zamanafzal i have updated description in changelog file and used `chore` instead, kindly check.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -17,6 +17,10 @@ Unreleased
 ----------
 * nothing unreleased
 
+[4.19.8]
+--------
+* chore: updated permission class ``IsInProvisioningAdminGroup`` for ``PendingEnterpriseCustomerAdminUser`` viewpoint and handled exceptions in serializer.
```
Comment (sameenfatima78) [inline CHANGELOG.rst:22]: this isn't a chore commit.

**Review by sameenfatima78 — COMMENTED** [review]
Have we tested this API via browser?

File: CHANGELOG.rst (hunk 2 above)
Comment (MueezKhan246) [reply CHANGELOG.rst:22]: @sameenfatima78 feedback was to avoid using `feat` for this.

**Comment by MueezKhan246** [comment]
> Have we tested this API via browser?

@sameenfatima78 in swagger?

**Pull request merged**